apps/src/templates/projects/projectsRedux.js

```diff
@@ -398,3 +423,16 @@ export const saveProjectName = (projectId, updatedName) => {
     });
   };
 };
+
+export const remix = (projectId) => {
+  return (dispatch) => {
+    fetchProjectToUpdate(projectId,
+    (error, data) => {
+      if (error) {
+        console.error(error);
+      } else {
+        dispatch(remixProjectOnServer(data));
```

Comment:
once you get your request to stop following the redirect, I'd recommend you `return dispatch(remixProjectOnServer(data))` (a `Promise`) here and then have `remixProjectOnServer` return the new channel id.